### ISSUE TITLE
[BUG] AXSellwands EconomyGuiShop Bug

### ISSUE THREAD

**Issue by qwerio** [Artillex-Studios/Issues#12]
### Description of the bug

AXSell Wands Bug

When using EconomyGuiShop, and item does not have sell value, when inspecting the total sell price of a chest is negative - negative with the count of items in the chest. (I would assume that when there is no sell value the count of the chest is multiplied by -1, rather than 0, since items that have no sell value are counted as sellable items)

### Expected behaviour

To display that there are no items to sell.

### Steps to reproduce

Create item without sell price. And try to inspect chest full with this item.

### Which plugin is this issue with?

axsellwands

### Versions

1.6.7

**Comment by BenceX100** [comment]
Can you check the 1.6.8 version? It should be fixed.

**Comment by qwerio** [comment]
Yeah this problems seems correct, but there is other strange behaviors.

When having only one unsellable item in the chest and X amount of sellable items it shows the the container has the correct amount of items. 
But if we have more than 1 stacked items (must be stacked in one slot, if we have on each slot 1 unsellable item it does not count) that are unsellable, and also have more than 0 sellable items in the container, when inspecting the amount of the stacked unsellable items are displayed towards the total sellable items in the chest. Also the displayed price is removed from the correct price.

Say we have wool sellable for 50$, and unsellable dirt.
If we have 1 dirt and any amount of wool - works fine.
If we have 2 dirt(stacked) and 1 amount of wool - it shows that the container has 3 items to sell for the price of 48$.(counting all items in container and removing the amount of the unsellable items from the sell price of the wool)
If we have 50 or more dirt and 1 wool - it displays that there is nothing to sell (this should be because the price of the dirt is -50 and combined with the price of the wool it equals 0, so I guess that's your check to not show if the chest has sellable items).

I was confused at the start, but as I typed the previous sentence I can conclude that the problem now is - when having more amount of stacked items than the sell price of the sellable items in the container - it displays that there is nothing sellable in the container.

*Note: It's just display bug, when selling items the money added are correct, as you may expect it to be.

**Comment by BenceX100** [comment]
I hope that this is fine by now, but if this still happens, let us know.
